project-setup/templates/README-template.md

```diff
@@ -0,0 +1,73 @@
+# Insert the name of your project here
+
+## Mission
+
+This should contain a brief (3-5 sentences) description of the mission of your
+project. The goal is to state what you are planning to work on and help external
+contributors understand roughly which types of features will likely be welcome
+for this project.
+
+See also [mission statement
+chapter](https://producingoss.com/en/producingoss.html#mission-statement) in
+Producing Open Source Software.
+
+## Getting Started
+
+This section should contain brief documentation written for first time users on
+how to get started using the project. Further more detailed documentation can be
+linked to from here.
+
+## Getting involved
+
+This section should include information on how to get in touch with the project:
+Typically this will contain links to archived, searchable and linkable
+communication channels.
+
+## Who we are
+
+This is a good place to give credit to Trusted Committers of the project.
+
+It's also a good place to include information on what being a Trusted Committer
+means for this project - although ideally all projects in an organisation use
+the same definition that is only linked to from here. The reason to keep the
+link here is for colleagues who have no or little experience with working in and
+contributing to InnerSource projects to have a direct link back to company wide
+information from the technological projects they need for their daily work.
+
+### How to become a Trusted Committer
```

Comment:
These sections on trusted committer management seem less important than contributing guides - suggest they go under the **Contributing** section.